docs/tools/kompose/user-guide.md

```diff
@@ -0,0 +1,307 @@
+
+```console
+$ kompose convert
+file "redis-svc.json" created
+file "web-svc.json" created
+file "redis-deployment.json" created
+file "web-deployment.json" created
+```
+The `*-deployment.json` files contain the Deployment objects.
+
+```console
+$ kompose convert --rc -y
+file "redis-svc.yaml" created
+file "web-svc.yaml" created
+file "redis-rc.yaml" created
+file "web-rc.yaml" created
+```
+
+The `*-rc.yaml` files contain the Replication Controller objects. If you want to specify replicas (default is 1), use `--replicas` flag: `$ kompose convert --rc --replicas 3 -y`
+
+```console
+$ kompose convert --ds -y
+file "redis-svc.yaml" created
+file "web-svc.yaml" created
+file "redis-daemonset.yaml" created
+file "web-daemonset.yaml" created
+```
+
+The `*-daemonset.yaml` files contain the Daemon Set objects
+
+If you want to generate a Chart to be used with [Helm](https://github.com/kubernetes/helm) simply do:
+
+```console
+$ kompose convert -c -y
+file "web-svc.yaml" created
+file "redis-svc.yaml" created
+file "web-deployment.yaml" created
+file "redis-deployment.yaml" created
+chart created in "./docker-compose/"
+
+$ tree docker-compose/
+docker-compose
+├── Chart.yaml
+├── README.md
+└── templates
+    ├── redis-deployment.yaml
+    ├── redis-svc.yaml
+    ├── web-deployment.yaml
+    └── web-svc.yaml
+```
+
+The chart structure is aimed at providing a skeleton for building your Helm charts.
+
+## Unsupported docker-compose configuration options
+
+Currently `kompose` does not support the following Docker Compose options.
+
+```
+"build", "cgroup_parent", "devices", "depends_on", "dns", "dns_search", "domainname", "env_file", "extends", "external_links", "extra_hosts", "hostname", "ipc", "logging", "mac_address", "mem_limit", "memswap_limit", "network_mode", "networks", "pid", "security_opt", "shm_size", "stop_signal", "volume_driver", "uts", "read_only", "stdin_open", "tty", "user", "ulimits", "dockerfile", "net"
+```
+
+For example:
+
+```console
+$ cat nginx.yml
+nginx:
+  image: nginx
+  dockerfile: foobar
+  build: ./foobar
+  cap_add:
+    - ALL
+  container_name: foobar
+
+$ kompose -f nginx.yml convert
+WARN[0000] Unsupported key build - ignoring             
+WARN[0000] Unsupported key cap_add - ignoring           
+WARN[0000] Unsupported key dockerfile - ignoring
+```
+
+## Labels
+
+`kompose` supports Kompose-specific labels within the `docker-compose.yml` file in order to explicitly imply a service type upon conversion.
+
+The currently supported options are:
+
+| Key                  | Value                               |
+|----------------------|-------------------------------------|
+| kompose.service.type | nodeport / clusterip / loadbalancer |
+
+
+For example:
```

Comment:
Somehow this part seems unfinished. Maybe change "for example" to "Here's an example docker-compose.yml file that ..."